src/frontend/pages/thank-you.tsx

```diff
@@ -1,5 +1,16 @@
 import { ReactElement } from 'react';
+import styles from '../styles/ThankYou.module.scss';
 
-export default function ThankYou(): ReactElement {
-  return <div>whatever</div>;
+export default function TimeOut(): ReactElement {
+  return (
+    <div className={styles.messageContainer}>
+      <div className={styles.timeOutMessage}>
+        Unfortunately, we were unable to find other players to begin the game at
+        this time.
+      </div>
+      <div className={styles.timeOutMessage}>
+        Please return to Qualtrics and try again.
```

Comment:
I don't think we want this "try again" part.

Reply:
Also do they need to return to qualtrics if they don't end up playing the game